SEO/BLL/CrawlManager.cs

```diff
@@ -30,13 +30,22 @@ public class CrawlManager
                         foreach (var docIps in docCrawl.IPs)
                         {
                             if (visitInfo.IPAddress.StartsWith(docIps.Address))
```

Comment:
we should find a way how to escape transferring data over the wire and do the filtering inside the database engine